.devcontainer/docker-compose.yml

```diff
@@ -4,6 +4,7 @@ volumes:
   home:
   workspaces:
   postgres-data:
+  minio-storage:
```

Comment:
let's call this `minio-data` for consistency

Reply:
yep. done!